README.md

```diff
@@ -13,7 +13,7 @@ React
 
```

Comment:
- [OPTIONAL] Your current Readme looks like this 👇  below. Let's make it look more orderly and clear. For that, you can put a dash(`-`) before any line of text that you want to start from a new line and separate from other texts. For example, for technology names in the `Built with` section can be like : `- HTML` etc
![Screenshot (130)](https://user-images.githubusercontent.com/67087664/148529048-ecd66166-4887-45cb-b4e0-5adaaff59ddf.png)